src/app/shell/details/side-menu/actions/actions.component.html

```diff
@@ -1,11 +1,9 @@
 <div [ngClass]="(isMobileScreen$ | async) ? 'container container-mobile' : 'container'" fxLayout="column" fxLayoutAlign="center center" fxLayoutGap='1.5rem'>
   <h4 class="card-title" *ngIf="workshop.price" fxFlexAlign="center"> {{ workshop.price }} грн/{{ PayRateTypeUkr[workshop.payRate] }} </h4>
   <h4 class="card-title" *ngIf="!workshop.price"> Безкоштовний гурток </h4>
-  <a [routerLink]="['/create-application', workshop.id]" (click)="login()">
-    <button mat-raised-button class="button button-application" fxLayout="row" fxLayoutAlign="center center">
-      ПОДАТИ ЗАЯВУ
+    <button mat-raised-button class="button button-application" fxLayout="row" fxLayoutAlign="center center" [disabled]="workshop?.status === workhopStatus.Closed">
+      <a [routerLink]="workshop?.status === workhopStatus.Closed ? null : ['/create-application', workshop.id]" (click)="login()">ПОДАТИ ЗАЯВУ</a>
```

Comment:
![image](https://user-images.githubusercontent.com/27493959/186134076-007240fb-0311-4e30-88ab-c28aca392a3d.png)
and move please check for workshop status to ngOnInit

Reply:
fixed